include/exadg/convection_diffusion/time_integration/time_int_bdf.cpp

```diff
@@ -557,7 +557,9 @@ TimeIntBDF<dim, Number>::postprocessing() const
   if(this->param.ale_formulation and this->get_time_step_number() == 1 and
      not this->param.restarted_simulation)
   {
-    pde_operator->move_grid_and_update_dependent_data_structures(this->get_time());
+    pde_operator->move_grid(this->get_time());
```

Comment:
In comment https://github.com/exadg/exadg/pull/404#discussion_r1177448382, it has been proposed to let the time integrator directly operate on the grid motion object, which can now be addressed easily without affecting the update of MatrixFree or the update of spatial operators (such as the interior penalty parameter).